src/main/java/com/google/sps/servlets/ConvertLocationServlet.java

```diff
@@ -0,0 +1,51 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.URL;
+import java.net.URLConnection;
+
+@WebServlet("/convert")
+public class ConvertLocationServlet extends HttpServlet {
+
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        response.setContentType("text/html");
+        String lat = request.getParameter("lat");
+        String lng = request.getParameter("lng");
+        String apiKey = "AIzaSyDbEPugXWcqo1q6b-X_pd09a0Zaj3trDOw";
```

Comment:
Do not expose api keys like this.

https://cloud.google.com/docs/authentication/api-keys#securing_an_api_key

A solution by another intern team was to have "a two key system controlled by environment variables. Their actual key, which is committed in a .env file, is restricted to only be usable from their appengine app as an http referrer. They have a second key which is usable from localhost but is never committed. When they start up a session  for testing they just set that as the relevant environment variable. So they have a key that is publicly known but not publicly usable and one that is publicly usable but not publicly known, which we thought was probably sufficient."

Reply:
fixed in new branch called: hide-yo-keys